demos/sjtwo/ltc4150/source/main.cpp

```diff
@@ -15,12 +16,15 @@ int main()
   LOG_INFO("Ltc4150 application starting...");
   // Creating GPIO on pin 2.0
   sjsu::lpc40xx::Gpio tick_pin(2, 0);
+  // Pass the tick_pin GPIO to the gpio counter. We want to keep
```

Comment:
Add some new lines in here to separate the comments+lines

Reply:
Done.